kotlinx-coroutines-test/api/kotlinx-coroutines-test.api

```diff
@@ -56,12 +56,11 @@ public final class kotlinx/coroutines/test/TestDispatchers {
 }
 
 public abstract interface class kotlinx/coroutines/test/UncaughtExceptionCaptor {
-	public abstract fun cleanupTestCoroutines ()V
+	public abstract fun cleanupTestCoroutinesCaptor ()V
```

Comment:
Without this rename, JS IR compiler just fails, and the native compiler complains
```
Member inherits different @Throws filters from UncaughtExceptionCaptor, DelayController
```

Reply:
Worth reporting, because JS IR should ignore `@Throws`